wire-ios-request-strategy/Sources/Payloads/Processing/PayloadProcessing+ConversationTests.swift

```diff
@@ -841,6 +841,44 @@ class PayloadProcessing_ConversationTests: MessagingTestBase {
             XCTAssertEqual(mockEventProcessor.calls.wipeGroup.count, 0)
         }
     }
+    
+    func testUpdateOrCreate_withMLSSelfGroupEpoch0_callsMLSServiceJoinGroup() {
+        let mockMLS = internalTest_UpdateOrCreate_withMLSSelfGroupEpoch(epoch: 0)
+        // then
+        XCTAssertTrue(!mockMLS.calls.createSelfGroup.isEmpty)
```

Comment:
why not do it this way?

```suggestion
        XCTAssertFalse(mockMLS.calls.createSelfGroup.isEmpty)
```